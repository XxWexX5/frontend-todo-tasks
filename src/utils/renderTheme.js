import { render } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import theme from '../styles/theme-styles';

export const renderTheme = (children) =>
  render(<ThemeProvider theme={theme}>{children}</ThemeProvider>);
