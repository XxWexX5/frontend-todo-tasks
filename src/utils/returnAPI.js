export function returnAPI() {
  return process.env.REACT_APP_URL_API;
}
