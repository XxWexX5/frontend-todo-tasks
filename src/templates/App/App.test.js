import { render, screen } from '@testing-library/react';
import Home from '.';

import { renderTheme } from '../../utils/renderTheme';

test('renders learn react link', () => {
  const { debug } = renderTheme(<Home />);

  const headingContainer = screen.getByRole('heading', {
    name: 'Hello',
  }).parentElement;

  expect(headingContainer).toMatchSnapshot();
  expect(headingContainer).toHaveStyleRule('color', '#0F0');
});
