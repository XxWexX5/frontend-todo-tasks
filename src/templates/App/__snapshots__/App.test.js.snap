// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
.c0 {
  color: #0F0;
}

<div
  class="c0"
  color="#0F0"
>
  <h1>
    Hello
  </h1>
</div>
`;
